README.md

```diff
@@ -3,39 +3,20 @@ This is a OpenID Connect example, used as an assigment in the OAuth2 and OpenID
 
 ## Dependencies
 
-**python 2.x** (tested with python 2.7.10)
+**python 3.7 (tested with python 3.7.4)
 
-**OpenSSL 1.0** to be able to do modern TLS versions. Python together with 0.9.x has a bug that makes it impossible to select protocol in the handshake, so it cannot connect to servers that have disabled SSLv2.
+**OpenSSL 1.0** to be able to do modern TLS versions.
 
-Python dependencies can be installed by using PIP: `pip install -r requirements.txt`
 
 ## Assignment
-There is some blank spaces in the code, which you're supposed to fill in. 
 
-### client.py
-1. Fill in the missing data for the token request
-    When done, you should be able to get a token from the server
-
-2. Fill in the data for the refresh request
-    When done, you should be able to refresh the access token
-
-3. Fill in the data for the revoke request
-    When done, you should be able to revoke the access token
-
-### app.py
-4. Add the token to the api request
-    You should now be able to request data from 
-
-5. Add openid to the requested scopes, in configuration.
-    You should get an id_token from the token response
-
-6. Validate the id_token
-    Use the validator to validate the id_token
+TODO!
```

Comment:
todo what?

Reply:
Add in the assignments. I removed the headline instead, since we will hand it out in the workshop.